ds-collector/ds-collector

```diff
@@ -833,7 +838,7 @@ get_infos() {
     # XXX – can be optimised to do all nodes in a rack in parallel
     for host in ${cassandraNodes} ; do
       hostName=$host
-      if [[ "$hostName" != \#* ]] && [ "$hostName" ]; then
+      if [[ "$hostName" != \#* ]] && [ "$hostName" ] && [[ ! ${skipNodes[*]} =~ "$hostName" ]]; then
```

Comment:
This match is fuzzy in nature, if skip nodes is a partial name - it has the potential to match multiple nodes - this might be intended, but hidden feature of the skip node list. e.g. a skip nodes with just 'o' would match to the hostname "test-node-a" as well as "test-node-b"

Reply:
suggestion? 🤷🏻‍♀️

Reply:
just the following?
```
      if [[ "$hostName" != \#* ]] && [ "$hostName" ] && [[ ! ${skipNodes[*]} == "$hostName" ]]; then

```

Reply:
put spaces either side of both the hostname and the skipnodes - since you had split based on spaces, it would then only get a match if it was an exact match, since the scenario of just a skipnodes of letter 'o' would then be expanded to ' o ' - which doesnt match to ' test-node-a ' anymore.

Reply:
i haven't come up with a working solution yet…

Reply:
will this do it :  " ${skipNodes[*]} " =~ " ${hostName} "

Reply:
uffda. i was testing `~=` instead of `=~` :'(

Reply:
should be good now.